setup.py

```diff
@@ -36,7 +36,7 @@
             ]
         },
     install_requires=[
-        'OpenFisca-Core >= 4.1.5.dev1, < 5.0',
+        'OpenFisca-Core == 4.2.1.dev0',
```

Comment:
Running the yaml test work, but serving the api doesn't. This is the cache problem I mentioned earlier.